src/server.ts

```diff
@@ -25,14 +29,25 @@ app.post('/graphql', (req, res) => {
 			source: req.body.query,
 			operationName: req.body.operationName,
 			variableValues: req.body.variables,
-			contextValue: new Context(
-				'0x19B4ED919081720a753337546284bA7075F21fE6'
-			)
+			contextValue: context
 		},
 		res
 	);
 });
 
+
+/**
+  * account login or creation endpoint
+	* since context is not yet initialized we need an
+	* endpoint that will do that without requiring a proper context
+	*/
+app.post('/account',(req,res)=>{
+	accId = verify(req.header('authorization'));
+	if(accId !== null){
+		context = new Context(accId);
+	}
+});
+

```

Comment:
This code should go to app.port('/graphql' ...). We don't want to create any additional endpoints.

Queries, Mutations, and Subscriptions that don't require the context will not check for them. So if the context is not available, thats okay. Queries that do require them will check if the context is there. Take a look at all the resolvers, they all check the context. So which resolver will not check the context? Well createNewAccount resolver.